test/example/deployment.yaml

```diff
@@ -20,4 +20,13 @@ spec:
       containers:
       - name: example
         image: quay.io/quay/busybox
-        command: ["sh", "-c", "while true; do date; sleep 10; done"]
+        command:
+        - sh
+        - -c
+        - |
+          trap exit TERM
+          while true; do
+              date
+              sleep 10 &
```

Comment:
Will this really make it wait for 10 seconds? Can you post a log of date time stamps in the log?

Reply:
This run sleep in the background without waiting it, and next line wait for it.